### DISCUSSION TITLE
Logging into Anilist in a linux environment.

### DISCUSSION THREAD

**Discussion by bersilin-robert1609** [aleganza/akuse#42] [General]
How to login into anilist account in the development mode in the linux environment? I tried packing it for linux using `npm run dist:linux` but after installing the package using `dpkg` the app didnt launch. Any help would be appreciated. Thanks.

OS: Ubuntu 23.10 Dev

**Comment by Ameliacarlo72** [comment]
Une ligne: To log into AniList in a Linux environment, you can use a web browser to access the AniList website and log in with your account credentials.
[https://www.uneligne.ch/](https://www.uneligne.ch/)

**Comment by aleganza** [comment]
I need to fix electron deep linking for linux
